Code/EnginePlugins/OpenXRPlugin/OpenXRStartup.cpp

```diff
@@ -0,0 +1,40 @@
+#include <OpenXRPluginPCH.h>
+
+#include <OpenXRPlugin/OpenXRIncludes.h>
+#include <OpenXRPlugin/OpenXRSingleton.h>
+#include <Foundation/Configuration/Startup.h>
+#include <Foundation/IO/FileSystem/FileReader.h>
+#include <Foundation/IO/FileSystem/FileWriter.h>
+#include <Foundation/IO/OpenDdlReader.h>
+#include <Foundation/IO/OpenDdlUtils.h>
+#include <Foundation/IO/OpenDdlWriter.h>
+#include <GameEngine/GameApplication/GameApplication.h>
+
+// clang-format off
+EZ_BEGIN_SUBSYSTEM_DECLARATION(OpenXR, OpenXRPlugin)
+
+  BEGIN_SUBSYSTEM_DEPENDENCIES
+    "Foundation",
+    "Core"
+  END_SUBSYSTEM_DEPENDENCIES
+
+  ON_CORESYSTEMS_STARTUP
+  {
+  }
+
+  ON_CORESYSTEMS_SHUTDOWN
+  {
+  }
+
+  ON_HIGHLEVELSYSTEMS_STARTUP
+  {
+  }
+
+  ON_HIGHLEVELSYSTEMS_SHUTDOWN
+  {
+  }
```

Comment:
Not needed?

Reply:
Yep and I can't remember why I added it in the first place.